framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -3078,59 +3107,93 @@ public Set<AnnotationMirror> getWidenedAnnotations(
     }
 
     /**
-     * Returns the types of the two arguments to the BinaryTree, accounting for widening and
-     * unboxing if applicable.
+     * Returns the types of the two arguments to the BinaryTree. Please refer to {@link
+     * #binaryTreeArgTypes(TypeMirror, AnnotatedTypeMirror, AnnotatedTypeMirror)} )} for more
+     * details.
      *
      * @param node a binary tree
-     * @return the types of the two arguments
+     * @return the types of the two arguments (note that the underlying java types of both operands
```

Comment:
In the detailed documentation you list two special cases, but here (and in the other return docs) you only mention the second one. Is it worth highlighting only one of the two cases here?

Reply:
Improved the detailed documentation and removed all the "(note that the underlying java types ...)"

The navigation in Javadoc should be straightforward.